.travis.yml

```diff
@@ -1,3 +1,3 @@
- language: node_js
- node_js:
-   - "4.2"
+language: node_js
+node_js:
+  - '6'
```

Comment:
This is required so we can use modern JS in our code like `Array.prototype.includes` 😄